plutus-emulator/src/Wallet/Emulator/Types.hs

```diff
@@ -280,16 +284,21 @@ instance WalletAPI MockWallet where
         privK <- use ownPrivateKey
         pure (Crypto.sign (BSL.toStrict bs) privK)
 
-    createPaymentWithChange vl = do
+    updatePaymentWithChange vl (oldIns, changeTx) = do
```

Comment:
`changeTx` -> `changeOut`? It's not a transaction!

Reply:
This block could do with a few comments.

Reply:
Should I place the comments here, or on the type class?

Reply:
I meant in the code itself, since it's not 100% obvious what it's doing without careful reading of the code.

Reply:
Added comments.